docs/contributing/coding-style.md

```diff
@@ -603,3 +627,19 @@ We do not suggest to modify the generated code other than the `tests` variable,
         }
         // generated test code
     ```
+
+1. Unused fields
+
+    By default, the template provides `fields` structure to initialize object of the test target. 
+    
+    But in some cases, not all `fields` are needed, so please delete the unnecessary fields.
+
+    If the test case needs only `addr` of `fields`, please delete other fields.
```

Comment:
```suggestion
    For example the following struct and the corresponding function:
    
    ```golang
    type server struct {
        addr string
        port int
    }
    func (s *server) Addr() string {
        return s.addr
     }
    ```

    And the generated test code is:
    ```golang
    func Test_server_Addr(t *testing.T) {
        type fields struct {
            addr string
            port int
	}
	type want struct {
        // generated test code
    ```

    If the test case needs only `addr` of `fields`, please delete the unused field like `port`.

    ```golang
    func Test_server_Addr(t *testing.T) {
        type fields struct {
            addr string
            // port int   <-- this line should be deleted
	}
	type want struct {
        // generated test code
    ```

```